day10/Consumer.java

```diff
@@ -0,0 +1,23 @@
+package day10;
+
+public class Consumer extends Thread{
+    Company c;
+    Consumer(Company c){
+        this.c = c;
+    }
+    @Override
+    public void run() {
+        while(true){
+            try {
+                this.c.consume_item();
+            } catch (InterruptedException e) {
+                throw new RuntimeException(e);
+            }
+            try {
+                Thread.sleep(1000);
+            } catch (InterruptedException e) {
+                throw new RuntimeException(e);
```

Comment:
don't throw any Exception here, print the error here using `System.err.println()`